### PR TITLE
fix focused cell can end up below the header

### PR DESCRIPTION
**Pull request by LiKang6688**

### CODE REVIEW

File: src/table/TableBodyWrapper.jsx [hunk 1]
```diff
@@ -67,7 +67,7 @@ const TableBodyWrapper = ({
   }, []);
 
   return (
-    <TableBody className={classes.cellBase}>
+    <TableBody className={classes.cellBase} style={{ height: `${rootElement.clientHeight - 102}px` }}>
```
Comment (haxxmaxx) [inline src/table/TableBodyWrapper.jsx:70]: Why 102? Would be nice if you can get this in a class, but not in `cellBase` since it is not related to cell.

Comment (LiKang6688) [reply src/table/TableBodyWrapper.jsx:70]: the height of <thead> + the height of <TablePagination>, which are fixed value. Sure.

Comment (haxxmaxx) [reply src/table/TableBodyWrapper.jsx:70]: the height of what and what? 🙃

Comment (LiKang6688) [reply src/table/TableBodyWrapper.jsx]: 😂 sorry, I should preview it first. the height of` <TableHead>` + the height of` <TablePagination>`

Comment (haxxmaxx) [reply src/table/TableBodyWrapper.jsx]: OK, no problem. Well the height of the TableHead varies with the font size?

File: src/table/TableBodyWrapper.jsx [hunk 2]
```diff
@@ -45,7 +48,7 @@ const TableBodyWrapper = ({
 }) => {
   const { rows, columns } = tableData;
   const hoverEffect = layout.components?.[0]?.content?.hoverEffect;
-  const bodyStyle = useMemo(() => getBodyStyle(layout, theme), [layout, theme.name()]);
+  const bodyStyle = useMemo(() => getBodyStyle(layout, theme, rootElement), [layout, theme.name()]);
   const classes = useStyles(bodyStyle);
```
Comment (haxxmaxx) [inline src/table/TableBodyWrapper.jsx:52]: I'd say the `rootElement` is not part of `bodyStyle`. concat it when calling `useStyles` here instead.
`useStyles({ ...bodyStyles, rootElement.height })`

File: src/table/TableBodyWrapper.jsx (hunk 1 above)
Comment (LiKang6688) [reply src/table/TableBodyWrapper.jsx]: good point. I need to update accordingly.

File: src/table/TableBodyWrapper.jsx (hunk 2 above)
Comment (LiKang6688) [reply src/table/TableBodyWrapper.jsx]: sure

**Review by haxxmaxx — APPROVED** [review]

**Pull request merged**